packages/blade/src/components/Typography/Code/Code.tsx

```diff
@@ -0,0 +1,96 @@
+import styled from 'styled-components';
+import { BaseText } from '../BaseText';
+import Box from '~components/Box';
+import type { TextTypes } from '~tokens/theme/theme';
+import { getPlatformType } from '~utils';
+
+export type CodeProps = {
+  /**
+   * children only supports string.
+   *
+   * ```tsx
+   * // ❌ Bad
+   * <Code><Box>something</Box></Code>
+   * ```
+   *
+   *```tsx
+   * // ✅ Good
+   * <Code>something</Code>
+   * ```
+   */
+  children: string;
+  /**
+   * Decides the fontSize and padding of Code
+   *
+   * @default medium
+   */
+  size?: 'large' | 'medium';
+  /**
+   * Decides the visibility and color of Code
+   *
+   * @default subtle
+   */
+  type?: TextTypes;
+};
+
+type CodeContainerProps = {
+  size: CodeProps['size'];
+  children: React.ReactNode;
+};
+
+const platformType = getPlatformType();
+const isPlatformWeb = platformType === 'browser' || platformType === 'node';
+
+const CodeContainer = styled(Box)<CodeContainerProps>((props) => {
+  const padding =
+    props.size === 'large'
+      ? `${props.theme.spacing[0]}px ${props.theme.spacing[2]}px`
+      : `${props.theme.spacing[1]}px ${props.theme.spacing[3]}px`;
+  return {
+    padding,
+    backgroundColor: props.theme.colors.brand.gray[300],
+    borderRadius: props.theme.border.radius.medium,
+    display: isPlatformWeb ? 'inline-block' : undefined,
+  };
+});
+
+/**
+ * Code component can be used for displaying token, variable names, or inlined code snippets.
+ *
+ * ## Usage
+ *
+ * ### In Web
+ * In web, you can use it inside `Text` component or individually. The component is set to display `inline-block`
+ *
+ * ```tsx
+ * <Text>Lorem ipsum <Code>SENTRY_TOKEN</Code> normal text</Text>
+ * ```
+ *
+ * ### In React Native
+ *
+ * In React Native, you would have to align it using flex to make sure the Code and the surrounding text is correctly aligned
+ *
+ * ```tsx
+ *  <Box flexWrap="wrap" flexDirection="row" alignItems="flex-start">
+ *   <Text>Lorem ipsum </Text>
+ *   <Code>SENTRY_TOKEN</Code>
+ *   <Text> normal text</Text>
+ * </Box>
+ * ```
+ */
+const Code = ({ children, size = 'medium', type = 'subtle' }: CodeProps): JSX.Element => {
+  return (
+    <CodeContainer size={size} as={isPlatformWeb ? 'span' : undefined}>
+      <BaseText
+        color={`surface.text.${type ?? 'subtle'}.lowContrast`}
```

Comment:
Do we need to do `type ?? subtle`? 

We are already setting default value to type in the param.

Reply:
I did for TS. It was still thinking that value can be undefined 🤔

Reply:
Don't think it's throwing error, maybe try restarting the ts server?

Reply:
ohhh right my badd!! My earlier implementation was a bit different where there was in some nested component where it was throwing the error. You're right its not throwing error here. Will remove it.

Reply:
done